INSTRUCTIONS.native.md

```diff
@@ -0,0 +1,92 @@
+The following instructions are for setting up MSVC without docker.
+
+# FAQ
+
+## Does it run on Ubuntu 18.04 LTS?
+
+Yes, but the install scripts won't work because `msitools` is too old. You'll need to install either via Docker or on a real Ubuntu 20.04 LTS machine; and later copy paste the files under `/opt/msvc`
+
+## Does it work with CMake?
+
+Yes, but a custom version is needed or else CMake will complain that `/opt/msvc/bin/x64/cl` can't build a simple program.
+
+## Can it build Debug versions?
+
+Yes, but there may be troubles. From wine errors appearing in the logs/console to problems when trying to launch it because of missing debug redistributables.
+
+For the best experience build in Release mode
+
+## I get an "access denied" error when building RC via CMake.
+
+`Ninja` Generator has a problem when setting up RC. Use `Unix Makefile`s instead
+
+There is a [custom version of Ninja](https://github.com/mstorsjo/ninja/commits/cross-msvc) to deal with MSVC cross compiling but I haven't checked it to see if it fixes the problem.
+
+## Project gets rebuild from scratch every time
+
+This is a [known issue](https://github.com/mstorsjo/msvc-wine/issues/29)
+Apply this patch:
+
+```diff
+diff --git a/wrappers/cl b/wrappers/cl
+index 06702ba..ced5360 100755
+--- a/wrappers/cl
++++ b/wrappers/cl
+@@ -1,3 +1,3 @@
+ #!/bin/bash
+ . $(dirname $0)/msvcenv.sh
+-$(dirname $0)/wine-msvc.sh $BINDIR/cl.exe "$@"
++$(dirname $0)/wine-msvc.sh $BINDIR/cl.exe "$@" | sed "/Note: including file:/ s/\\\\/\//g" -
+```
+
+## fatal error C1902: Program database manager mismatch; please check your installation
+
+You need winbind: `sudo apt install winbind`
+
+Issue is [being tracked](https://github.com/mstorsjo/msvc-wine/issues/6).
+
+# Instructions
+
+## Prerequisites
+
+```bash
+apt-get update
+apt-get install -y wine64-development python msitools python-simplejson python-six ca-certificates winbind
+```
+
+## Installation
+
+We're going to install it into `~/my_msvc` to avoid needing root privileges on a non-contained system.
+You can change the location to wherever you want, even real `/opt/` if you want to deal with chown'ing all files to root before moving
+
+```bash
+# Install MSVC
+./vsdownload.py --dest ~/my_msvc/opt/msvc
+./install.sh ~/my_msvc/opt/msvc
```

Comment:
Here I think it'd be nice to have a separate comment for the `install.sh` line, singling them out more, e.g. like this:
```
# Download and unpack MSVC
./vsdownload.py ...
# Add wrapper scripts, do minor cleanup of the unpacked MSVC installation
./install.sh ...
```

Reply:
Done!